utils/utils.h

```diff
@@ -121,15 +121,11 @@
             pthread_mutex_unlock(&x);                                \
           } while (0)
 
-# define ERROR(fmt, ...)                                             \
-         do {                                                        \
-           fprintf(stderr, "Error: " fmt " [at %s+%d :<%s>]\n",      \
-                   __VA_ARGS__, __FILE__, __LINE__, __FUNCTION__);   \
-          } while (0)
-
-# define  MSG(fmt, ...)                                              \
+# define  MSG(fd, fmt, ...)                                          \
           do {                                                       \
-            fprintf(stdout, fmt, __VA_ARGS__);                       \
+            if (fd < 0)                                              \
+              fd = stderr;                                           \
+            fprintf(fd, fmt, __VA_ARGS__);                           \
```

Comment:
Here how about adding the __FILE__, __LINE__ and __FUNCTION__ in the log messages ?

Reply:
No these are dumped to console. For example, the create cli response output:
```
# gluster-block create block-test/sample-block ha 3 192.168.1.11,192.168.1.12,192.168.1.13 1GiB
IQN: iqn.2016-12.org.gluster-block:aafea465-9167-4880-b37c-2c36db8562ea
PORTAL(S): 192.168.1.11:3260 192.168.1.12:3260 192.168.1.13:3260
RESULT: SUCCESS
````

It doesn't make any sense to print the FILE, LINE and FUNCTION  in this case.

For error outputs, it might makesense in some scenarios, but we will get these info in the LOG's files.

Reply:
yeah, that's good.